dataactbroker/handlers/jobHandler.py

```diff
@@ -24,6 +24,10 @@ def getSubmissionById(self,submissionId):
         result = self.runUniqueQuery(query,"No submission with that ID","Multiple submissions with that ID")
         return result
 
+    def getSubmissionsByUserAgency(self,user,limit=5):
```

Comment:
@bsweger Either way would work just fine. I added the limit just as a parameter in anticipation of a pagination story where they could enter a specific number or get all of them at once. We could make the web app filter itself. I'm good with either way, it's a simple update in the backend. What would you suggest?